db/trie/mptrie/merklepatriciatrie_benchmark_test.go

```diff
@@ -2,89 +2,98 @@ package mptrie
 
 import (
 	"context"
+	"encoding/binary"
 	"fmt"
 	"testing"
 
-	"github.com/iotexproject/go-pkgs/hash"
+	"github.com/stretchr/testify/require"
+
 	"github.com/iotexproject/iotex-core/config"
 	"github.com/iotexproject/iotex-core/db"
 	"github.com/iotexproject/iotex-core/db/batch"
 	"github.com/iotexproject/iotex-core/db/trie"
 	"github.com/iotexproject/iotex-core/testutil"
-	"github.com/pkg/errors"
-	"github.com/stretchr/testify/require"
 )
 
-// new version
-func benchmarkTrie(async bool, b *testing.B) {
-	if testing.Short() {
-		b.Skip("Skipping TestPressure in short mode.")
-	}
-	require := require.New(b)
-	testPath, err := testutil.PathOfTempFile(fmt.Sprintf("test-kv-store-%t.bolt", async))
-	require.NoError(err)
-	cfg := config.Default.DB
-	cfg.DbPath = testPath
-	dao := db.NewBoltDB(cfg)
-	flusher, err := db.NewKVStoreFlusher(dao, batch.NewCachedBatch())
-	flusherKV := flusher.KVStoreWithBuffer()
-	kv, err := trie.NewKVStore("test", flusherKV)
-	require.NoError(err)
-	require.NoError(kv.Start(context.Background()))
-	var tr trie.Trie
+// BenchmarkTrieGet-4                	   29383	     44256 ns/op	   19754 B/op	     269 allocs/op
+// BenchmarkTrieGetWithAsync-4       	 1214578	      1022 ns/op	     160 B/op	       5 allocs/op
+// BenchmarkTrieGetDisk-4            	   26509	     49948 ns/op	   25371 B/op	     305 allocs/op
+// BenchmarkTrieGetDiskWithAsync-4   	 1156329	      1101 ns/op	     160 B/op	       5 allocs/op
+// BenchmarkUpsertLE-4               	   10000	    156839 ns/op	   41624 B/op	     434 allocs/op
+// BenchmarkUpsertBE-4               	    9362	    251458 ns/op	   79795 B/op	     900 allocs/op
```

Comment:
delete, b/c this is relying on your test env, which is not producible. You can leave this in the description of this PR.

Reply:
done